bool see_circle
geometry_msgs/Vector3 circle

geometry_msgs/Vector3[] pointT

geometry_msgs/Vector3[] pointL

geometry_msgs/Vector3[] goal_posts